packages/cli/src/commands/generate/page/__tests__/page.test.js

```diff
@@ -3,42 +3,45 @@ globalThis.__dirname = __dirname
 globalThis.mockFs = false
 let mockFiles = {}
 
-jest.mock('fs', () => {
-  const actual = jest.requireActual('fs')
+vi.mock('fs-extra', async (importOriginal) => {
+  const originalFsExtra = await importOriginal()
```

Comment:
I'm not trying to be difficult, I promise 🙃 Truly asking to better understand.
Here you decided to not keep the style of the existing test and instead go with the `importOriginal` callback parameter. Is there a difference? Or is it just up to personal preference in this case?

Reply:
And no need to change anything. I'm just curious 🙂

Reply:
Yeah, I guess I've been a little inconsistent in my application of the "copy exactly as it used to be" rule. In this case I've updated to the `importOriginal` - which is the way I'd want it to be. You have pointed out elsewhere where I didn't update it and kept the previous way. 

It's not a big deal I'm happy to switch out to consistently do `importOriginal` if you want?